package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bcgsc-pori/graphkb-client",
-  "version": "4.2.4",
+  "version": "5.0.0",
```

Comment:
will bump version when we do the release but not when merging to the dev branch

Reply:
oh, right I think this was because it's based off of the v5 client update, will change